PODMAN.md

```diff
@@ -2,22 +2,39 @@
 
 Install podman 3.1+, then run it in docker compatible mode:
 
+## Rootless podman
+
 ```
 systemctl start --user podman.socket
 ```
 
-After this command, you can use kubevirtci with the typical `make cluster-*`
-commands.
+Currently rootless podman is **not** working with the `make cluster-sync`
+command, essentially because incoming traffic is coming from the loopback device
+instead of eth0.
+
+The current rules - [0], [1] - allow `make cluster-up` to run successfully, but
+unfortunately they break the cluster's network connectivity in a subtle way:
+image pulling fails because outgoing traffic to ports 22 6443 8443 80 443 30007
+30008 31001 is redirected to the VM in the node01 container (i.e. itself)
```

Comment:
Maybe it is more accurate to say "redirected to the VM in the respective node container (i.e. itself)".

Reply:
Done. Good point. It read weird before.